test/modules/viewport.js

```diff
@@ -555,15 +557,15 @@
             var viewport = viewer.viewport;
 
             viewport.setFlip(true);
-            viewport.setRotation(45, true);
+            viewport.setRotation(45, null, true);
 
             viewport.fitBounds(new OpenSeadragon.Rect(1, 1, 1, 1), true);
             viewport.applyConstraints(true);
             var bounds = viewport.getBounds();
             Util.assertRectangleEquals(
                 assert,
+                new OpenSeadragon.Rect(1.2071067811865466, 0.20710678118654746, Math.sqrt(2), Math.sqrt(2), 45),
```

Comment:
@pearcetm Likewise here.

Reply:
See https://github.com/openseadragon/openseadragon/pull/2249 for fix